moveit_core/collision_detection_bullet/include/moveit/collision_detection_bullet/bullet_integration/bullet_utils.h

```diff
@@ -35,14 +35,16 @@
 #pragma once
 
 #include <btBulletCollisionCommon.h>
```

Comment:
This registers as a false-positive because bullet seems to use includes without prefix. Not much we can do about it I think, or do we want to add exceptions for specific header names?

Reply:
What do you mean by false-positive here?

Reply:
this includes bullet headers, but it's grouped with standard headers because there is no prefix used in the include path

Reply:
Got it. However, before the bullet header was not grouped with anything else as well...